examples/sample_entry.json

```diff
@@ -0,0 +1,23 @@
+{
+  "entries": [
+    {
+    "spiffe_id": {
+      "path": "/sample_path1"
+    },
+    "selectors": [
+        {
+          "type": "k8s",
+          "value": "ns:spire"
+        },
+        {
+          "type": "k8s",
+          "value": "aspire-k8s-registrar"
```

Comment:
Here we have a small typo. 

Let's create a file `sample_registrar.json` with a following content:


```{
  "entries": [
    {
    "spiffe_id": {
      "path": "/registrar1"
    },
    "selectors": [
        {
          "type": "k8s",
          "value": "ns:spire"
        },
        {
          "type": "k8s",
          "value": "sa:spire-k8s-registrar"
        },
        {
          "type": "k8s",
          "value": "container-name:k8s-workload-registrar"
        }
      ],
      "admin": true
    }
  ]
}
```

Reply:
Done!